homeassistant/components/homekit/accessories.py

```diff
@@ -137,6 +139,18 @@ def update_state(self, new_state):
         """
         raise NotImplementedError()
 
+    def call_service(self, domain, service, service_data, value=None):
+        """Fire event and call service for changes from HomeKit."""
+        event_data = {
+            ATTR_ENTITY_ID: self.entity_id,
+            ATTR_DISPLAY_NAME: self.display_name,
+            ATTR_SERVICE: service,
+            ATTR_VALUE: value
+        }
+        self.hass.bus.fire(EVENT_HOMEKIT_CHANGED, event_data)
+
```

Comment:
Probably super unnecessary, but can you move the empty line from here between `event_data` and `bus.fire`?

Reply:
Done. :-)